src/main/java/net/dries007/tfc/common/container/KnappingContainer.java

```diff
@@ -169,12 +169,7 @@ public boolean isItemValid(int slot, ItemStack stack)
     @Override
     protected boolean moveStack(ItemStack stack, int slotIndex)
     {
-        return switch (typeOf(slotIndex))
-            {
-                case CONTAINER -> !moveItemStackTo(stack, containerSlots, containerSlots + 36, true); // Hotbar first
-                case HOTBAR -> !moveItemStackTo(stack, containerSlots, containerSlots + 27, false);
-                case MAIN_INVENTORY -> !moveItemStackTo(stack, containerSlots + 27, containerSlots + 36, false);
-            };
+        return true; // no shift clicking
```

Comment:
Can this not be modified to ignore the specified slot only? The slot ranges here are specific slots to transfer to. Ignoring shift clicks entirely seems a bit slapdash (and also a bit of a "remove the bug in the feature by removing the feature" fix)